duckduckgo/Android
# Update DDG logo asset in Dark theme

Task/Issue URL: https://app.asana.com/0/488551667048375/1205121872781458/f

### Description
Update DDG logo asset in Dark theme
### Steps to test this PR

_System Theme_
- Install from this branch
- Set system theme to Dark
- Open a new tab without any favorites saved
- [ ] Check logo title is white
- Set system theme to Light
- [ ] Check logo title is changed to "black"

_DDG Theme_
- Install from this branch
- Set system theme to Light
- go to Settings > Appearance > Theme
- Select Dark
- Open a new tab without any favorites saved
- [ ] Check logo title is white

### UI changes
| Before  | After |
| ------ | ----- |
![Screenshot_20230803_123916_DuckDuckGo](https://github.com/duckduckgo/Android/assets/20798495/d7715d00-8106-46af-9272-6c6a41a49242)|![Screenshot_20230803_130956_DuckDuckGo](https://github.com/duckduckgo/Android/assets/20798495/2de645f7-8e09-4cc1-bfcb-f88b80ca59cc)|
